src/callbacks_step/analysis_dg2d_parallel.jl

```diff
@@ -5,8 +5,8 @@
 @muladd begin
 
 
-function calc_error_norms(func, u, t, analyzer,
-                          mesh::ParallelTreeMesh{2}, equations, initial_condition,
+function calc_error_norms(func, u::TrixiMPIArray, t, analyzer,
```

Comment:
As per our discussion somewhere else, didn't we want to dispatch on the parallel tree type where possible?

Reply:
Apples to multiple functions in this file and also for 3D

Reply:
efa08e7a